src/Apps/Order/Components/AddressVerificationFlow.tsx

```diff
@@ -66,7 +78,30 @@ enum AddressSuggestionRadioButton {
   user_address = "What you entered",
 }
 
+const fallbackFromFormValues = (address: AddressValues): AddressOption => {
+  return {
+    key: USER_INPUT,
+    recommended: false,
+    lines: [
+      address.addressLine1,
+      address.addressLine2 || null,
+      `${address.city}, ${address.region} ${address.postalCode}`,
```

Comment:
This could be more clever (array with filter/join plus the comma thing) but given it's an error case I'm inclined to leave it for now.

Reply:
Clever code is usually more expensive to read and change, so I'm pretty happy with this :)